### DISCUSSION TITLE
关于最大数据库连接数

### DISCUSSION THREAD

**Discussion by limfoo** [teableio/teable#727] [Q&A] [answered]
请问数据库最大连接数只有3的情况下大家是怎么把数据接入Power BI/Power Query的？一两个查询就超过3个连接数导致数据源刷新失败了呀。还是我使用的方法不对？

**Comment by tea-artist** [comment, marked as answer]
You can change the `DEFAULT_MAX_BASE_DB_CONNECTIONS` environment variable to any value you want.
We will be making some adjustments soon, we will no longer set a default maximum number of connections when self-deployed.

**Reply by limfoo** [reply]
Well noted, thanks.
